tiup/tiup-cluster.md

```diff
@@ -389,6 +389,39 @@ tiup cluster reload prod-cluster
 
 The command sends the configuration to the target machine and restarts the cluster to make the configuration take effect.
 
+> **Note:**
+>
+> For monitoring components, customize the configuration by executing the `tiup cluster edit-config` command to add a custom configuration path on the corresponding instance. For example:
+
+```yaml
+---
+grafana_servers:
+  - host: 172.16.5.134
+    dashboard_dir: /path/to/local/dashboards/dir
+monitoring_servers:
+  - host: 172.16.5.134
+    rule_dir: /path/to/local/rules/dir
+alertmanager_servers:
+  - host: 172.16.5.134
+    config_file: /path/to/local/alertmanager.yml
+```
+
+The content and format requirements for files under the specified path are as follows:
+
+- The folder specified in the `dashboard_dir` field of `grafana_servers` need to contain full `*.json` files
+- The folder specified in the `rule_dir` field of `monitoring_servers` need to contain full `*.rules.yml` files
+- For the format of files specified in the `config_file` field of `alertmanager_servers`, refer to [this template](https://github.com/pingcap/tiup/blob/master/templates/config/alertmanager.yml)
+
+When you execute `tiup reload`, TiUP first deletes all old configuration files in the target machine and then uploads the corresponding configuration from the control machine to the corresponding configuration directory of the target machine. Therefore, if you want to modify a particular configuration file, make sure that all configuration files (including the unmodified ones) are in the same directory. For example, to modify Grafana's `tidb.json` file, you need to first copy all the `*.json` files from Grafana's `dashboards` directory to your local directory. Otherwise, other JSON files will be missing from the target machine.
+
+> **Note:**
+>
+> If you have configured the `dashboard_dir` field of `grafana_servers`, complete the following operations when executing the `tiup cluster rename` command to rename the cluster:
+>
+> 1. `dashboards` in the local directory contains cluster name information. Change the cluster name in the local `dashboards` to the new cluster name.
```

Comment:
Agree with your suggested version. 👍🏻  But could I say "Change the cluster name to the new cluster name in the local `dashboards` directory"? Because the next two items of this list start with a verb, so I think it would be better to unify them. @lilin90

Reply:
Yes, it would be better to keep them consistent. It would be more friendly to first tell the place and then the action. So I added another suggestion for step 2.

Reply:
Get that. Thanks for you suggestion. 😊